lib/recurly/request.php

```diff
@@ -55,6 +57,16 @@ public function getBody(): ?array
         return $this->_body;
     }
 
+    /**
+     * Returns the JSON body included in the request
+     * 
+     * @return array The request body
+     */
+    public function getJson(): ?string
```

Comment:
Any opposition to calling this `getBodyAsJson`?

Reply:
Nope, will update